src/library/scala/collection/immutable/Range.scala

```diff
@@ -401,18 +401,12 @@ extends scala.collection.AbstractSeq[Int]
    *  equals contract.
    */
 
-  override def toString() = {
-    val endStr =
-      if (numRangeElements > Range.MAX_PRINT || (!isEmpty && numRangeElements < 0)) ", ... )" else ")"
-    take(Range.MAX_PRINT).mkString("Range(", ", ", endStr)
-  }
+  override def toString = s"Range($start, $end, $step)"
```

Comment:
This does not take `isInclusive` into account.
I suggest delegating to the `description` method, which already implements a nice string representation for ranges.